src/Models/Article.php

```diff
@@ -110,4 +114,16 @@ public function category()
     {
         return $this->belongsTo(Category::class, 'category_id', 'id');
     }
+
+    public function getMedia(): array
+    {
+        return $this->featured_image ?: [];
+    }
```

Comment:
I believe this is not needed according to the [docs](https://laravel-strata.com/docs/media/models/).

```suggestion
```